cypress/support/shouldSmokeTest/index.test.js

```diff
@@ -0,0 +1,23 @@
+import config from '../config/services';
+import shouldSmokeTest from './index';
+
+const PAGE_TYPE = 'frontPage';
+const SERVICE = 'korean';
+
+it('should return smoke value from config when env variable is true', () => {
+  global.Cypress.env = jest.fn().mockReturnValue(true);
```

Comment:
Should `env` be `true`? I thought `Cypress.env` was an object ... https://github.com/bbc/simorgh/blob/latest/cypress.json#L4-L8

Reply:
`Cypress.env` appears to be a function because it's used like `Cypress.env('SMOKE')`

Reply:
Gotcha, can we reset the mock after each test please